src/ngraph_encapsulate_op.cc

```diff
@@ -240,6 +256,7 @@ class NGraphEncapsulateOp : public OpKernel {
     ng::runtime::Backend* op_backend =
         BackendManager::GetBackend(m_op_backend_name);
 
+    Event event_func_maybe_create("FunctionMaybeCreate", name().c_str());
```

Comment:
We could separate the "cache lookup" vs "TranslateGraph + compile" time and have 2 separate events for them (we could separate TranslateGraph and ngraph-compile as well, to get an idea which part of the function compilation takes more time)
OR
`m_args` could be set for event `event_func_maybe_create`, to say if its a cache hit or a cache miss.
Then we wouldn't have to look at the length of the `event_func_maybe_create` event and guess if there was a cache hit or miss

We could do it in a separate PR

Reply:
We will address it in a separate PR.